contracts/FungibleToken.cdc

```diff
@@ -77,6 +77,13 @@ access(all) contract interface FungibleToken: ViewResolver {
     ///
     access(all) resource interface Balance {
         access(all) var balance: UFix64
+
+        // This default implementation needs to be in a separate interface
+        // from the one in `Vault` so that the conditions get enforced
+        // in the correct one
+        access(contract) fun burnCallback() {
```

Comment:
```suggestion
        /// This default implementation needs to be in a separate interface
        /// from the one in `Vault` so that the conditions get enforced
        /// in the correct one.
        /// Note: Anyone implementing `Balance` without also implementing
        /// `Vault` should also inherit `Burner.Burnable` for this method to
        /// be executed when the resource is passed to `Burner.burn()`
        access(contract) fun burnCallback() {
```